VariantEncoding.md

```diff
@@ -386,11 +386,15 @@ The Decimal type contains a scale, but no precision. The implied precision of a
 | Exact Numeric        | decimal8                    | `9`     | DECIMAL(precision, scale)   | 1 byte scale in range [0, 38], followed by little-endian unscaled value (see decimal table)                         |
 | Exact Numeric        | decimal16                   | `10`    | DECIMAL(precision, scale)   | 1 byte scale in range [0, 38], followed by little-endian unscaled value (see decimal table)                         |
 | Date                 | date                        | `11`    | DATE                        | 4 byte little-endian                                                                                                |
-| Timestamp            | timestamp                   | `12`    | TIMESTAMP(true, MICROS)     | 8-byte little-endian                                                                                                |
-| TimestampNTZ         | timestamp without time zone | `13`    | TIMESTAMP(false, MICROS)    | 8-byte little-endian                                                                                                |
+| Timestamp            | timestamp with time zone    | `12`    | TIMESTAMP(isAdjustedToUTC=true, MICROS)     | 8-byte little-endian                                                                                                |
```

Comment:
I assume some of these changes were weren't' intended, these lines shouldn't be changed

Reply:
AH nvm, I see what's going on here

Reply:
Actually maybe I don't, "timestamp" isn't a physical type in parquet correct?

Reply:
There is no Date or timestamp for physical type. I'm wondering if this column should mean `type category`. @gene-db ?

Reply:
I wonder if we should just eliminate the first column (and annotate logical type with additional information like exact numeric).